CHANGELOG.md

```diff
@@ -1,20 +1,19 @@
-## x.x.x (Month Day, Year)
+## 0.15.1 (November 11, 2016)
```

Comment:
Could you leave the existing changelog lines as-is and just add your changes to the relevant Summary, Features, etc. sections? The version, release date, and existing entries in the changelog should be the same as they were.